File: ucare_backend/src/main/java/com/douzone/ucare/service/StatusService.java
package com.douzone.ucare.service;

import java.util.List;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import com.douzone.ucare.repository.StatusRepository;
import com.douzone.ucare.vo.MedicineVo;
import com.douzone.ucare.vo.ReceiptVo;

@Service
public class StatusService {
	
	@Autowired
	private StatusRepository statusRepository;

	public List<ReceiptVo> retrieveStatus(String date) {
		return statusRepository.retrieveStatus(date);
	}

	public int update(ReceiptVo data) {
		return statusRepository.update(data);
	}

	public int delete(Long receiptNo) {
		return statusRepository.delete(receiptNo);
	}
}

File: ucare_frontend/src/service/statusService.js
import axios from 'axios';
import { STATUS_API_BASE_URL } from './urlConfig'

class statusService {
  retrieve(date){
    return axios.get(STATUS_API_BASE_URL + '/retrieve/' + date);
  }
  update(data){
    return axios.put(STATUS_API_BASE_URL + '/update', data)
  }
  delete(data){
    return axios.delete(STATUS_API_BASE_URL + '/delete/' + data);
  }
}

export default new statusService();